dojo/jira_link/views.py

```diff
@@ -136,7 +136,7 @@ def webhook(request, secret=None):
                 jira_usernames = JIRA_Instance.objects.values_list('username', flat=True)
                 for jira_userid in jira_usernames:
                     if jira_userid.lower() in commentor.lower():
-                        logger.debug('skipping incoming JIRA comment as the user id of the comment mathces the JIRA user in Defect Dojo')
+                        logger.info('skipping incoming JIRA comment as the user id of the comment mathces the JIRA user in Defect Dojo')
```

Comment:
I think this should remain at DEBUG because it wil happen for every note we put into defect dojo

Reply:
I disagree. This is important information I didn't know about, and luckily I knew how to turn the debugger on to figure out what the issue is.

Reply:
but it will then log a line every time somebody puts a note in defect dojo for findings with a jira attached.

Reply:
No. Just when commenting on your own tickets. Don't you use service users?

Reply:
I have a user in jira with username='defect.dojo'. Whenever somebody adds a note in dojo and pushes it to JIRA, there will be a comment in JIRA under that username. So author=='defect.dojo'. This comment triggers a webhook and the same comment gets sent to dojo. The author of this comment in jira is 'defect.dojo', so that's what gets sent in the as author/commenter in the webhook payload. Dojo sees this webhook coming in and sees that the commenter is 'defect.dojo'. Which is equal to the jira username for the dojo service user in JIRA. So it ignores the comment because it is the actual comment added to dojo that triggered the webhook. This happens for every note added to dojo which is pushed to JIRA. So it is normal beheviour and should be logged at most at DEBUG level.